src/pages/api/github/newBranch.ts

```diff
@@ -1,10 +1,10 @@
 import { upstreamOwner } from "@/config/default";
-import { Octokit } from "@octokit/core";
+import { Octokit } from "@octokit/rest";
```

Comment:
is this library backwards compatible with `@octokit/core`?

Reply:
Yes, the `@octokit/rest` library extends `@octokit/core` by adding methods for common GitHub REST API endpoints, while @octokit/core is a lower-level, more flexible client.

 `@octokit/rest` was needed for the GitHub App authentication, and then I had type issues with the rest of octokit usage, that's why I also switched to that in the rest of the files.